bucket/checksum-compare.json

```diff
@@ -0,0 +1,27 @@
+{
+    "version": "1.42",
+    "description": "Calculates and checks SHA1/MD5 checksums.",
+    "license": "Freeware",
+    "homepage": "http://checksumcompare.sanktuaire.com",
+    "hash": "e13ff69013a554bb8c33e428783e860f5ae0567db7db13aab323cb0043900547",
+    "url": "http://checksumcompare.sanktuaire.com/download.php?file=portable&version=v1.42#/dl.7z",
+    "bin": [
+        [
+            "Checksum compare.exe",
+            "checksum-compare"
+        ]
+    ],
+    "shortcuts": [
+        [
+            "Checksum compare.exe",
+            "Checksum compare"
+        ]
+    ],
+    "checkver": {
+        "url": "http://checksumcompare.sanktuaire.com/downloads",
+        "regex": "\\(Version Actuelle : v([\\d.]+)\\)"
+    },
+    "autoupdate": {
+        "url": "http://checksumcompare.sanktuaire.com/download.php?file=portable&version=v$version"
```

Comment:
```suggestion
        "url": "http://sanktuaire.free.fr/checksumcompare/Checksum-compare-v$version.7z",
        "hash": {
            "url": "http://checksumcompare.sanktuaire.com/downloads-en",
            "regex": "(?sm)$basename.*?$sha1"
        }
```

Reply:
Why this change when the original URL is from the official website?

Reply:
This is from oficial page too... Why would you have URL with redirect, when you can have static, direct URL??

Reply:
![image](https://user-images.githubusercontent.com/460971/90601815-1134ca00-e1f9-11ea-8dd6-35b77bc81b4a.png)

It does not redirect me or maybe I did not notice.

Reply:
... 


![image](https://user-images.githubusercontent.com/13260377/90602000-51944800-e1f9-11ea-9405-7b44083e1ee3.png)

Reply:
Okay, no problem. Cannot commit the suggestion now though as it is outdated.